### PR TITLE
fly to places

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Center on caller location and fly to zone when clicked

### CODE REVIEW

File: web/src/features/map/Map.tsx [hunk 1]
```diff
@@ -232,6 +235,10 @@ export default function MapPage(): ReactElement {
   };
 
   const onLoad = () => {
+    if (data?.callerLocation) {
+      const map = mapReference.current?.getMap();
+      map?.flyTo({ center: [data?.callerLocation[0], data?.callerLocation[1]] });
```
Comment (madsnedergaard) [inline web/src/features/map/Map.tsx:240]: Nitpick, as the if-check above should ensure that we have something here :)
```suggestion
      map?.flyTo({ center: [data.callerLocation[0], data.callerLocation[1]] });
```

File: web/src/features/map/Map.tsx [hunk 2]
```diff
@@ -232,6 +235,10 @@ export default function MapPage(): ReactElement {
   };
 
   const onLoad = () => {
+    if (data?.callerLocation) {
+      const map = mapReference.current?.getMap();
+      map?.flyTo({ center: [data.callerLocation[0], data.callerLocation[1]] });
+    }
     setIsLoadingMap(false);
   };
```
Comment (madsnedergaard) [inline web/src/features/map/Map.tsx:243]: Hmm actually, right now it's only "flying" after map is loaded, but what if we run this in a useEffect instead, so it's triggered as soon as the callerLocation is available? :)

Comment (tonypls) [reply web/src/features/map/Map.tsx]: Changes added!

File: web/src/features/map/Map.tsx [hunk 3]
```diff
@@ -132,6 +132,9 @@ export default function MapPage(): ReactElement {
         );
       }
     }
+    if (data?.callerLocation) {
+      map.flyTo({ center: [data.callerLocation[0], data.callerLocation[1]] });
+    }
```
Comment (madsnedergaard) [inline web/src/features/map/Map.tsx:137]: Having it inside this useEffect means it jumps every time the timeslider is changed - which is a kinda weird experience :D 

https://user-images.githubusercontent.com/3296643/210750020-6a7a0d02-e321-4b7f-a2d4-424896b2e40b.mp4


Could you add it to a new useEffect that only happens on initial load (so no dependencies)?

Comment (tonypls) [reply web/src/features/map/Map.tsx:137]: > Having it inside this useEffect means it jumps every time the timeslider is changed - which is a kinda weird experience :D
> 
>  Kapture.2023-01-05.at.10.44.34.mp4 
> Could you add it to a new useEffect that only happens on initial load (so no dependencies)?

Entirely intended feature 😆

**Pull request merged**